.github/labeler.yml

```diff
@@ -39,22 +39,18 @@
   - any-glob-to-any-file:
     - "target/linux/bcm4908/**"
     - "package/boot/uboot-bcm4908/**"
+    - "package/boot/arm-trusted-firmware-bcm63xx/**"
 "target/bcm53xx":
```

Comment:
Why is the firmware path moved to another target?

Reply:
I think it was added to the wrong target from the beginning. it is named `bcm63xx`, but it is for the `bcm4908` target.